reactor-core/src/main/java/reactor/core/publisher/FluxMetricsFuseable.java

```diff
@@ -185,4 +183,10 @@ public int size() {
 			return qs == null ? 0 : qs.size();
 		}
 	}
+
+	@Override
+	public Object scanUnsafe(Attr key) {
```

Comment:
move up, close to `subscribeOrReturn`. I was first confused that this was covering the inner (which is already covered by virtue of extending the non=fused version)